docs/guide/ffi_overview.md

```
@@ -0,0 +1,1067 @@
+uint64_t wallet_burn_tari(
+    TariWallet* wallet,
+    uint64_t amount,               // Amount to burn
+    uint64_t fee_per_gram,         // Fee rate
+    char* message,                 // Burn message
+    int* error_out
+);
+```
+
+#### Transaction Queries
+
+**wallet_get_completed_transactions**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)